importer/Uploader.py

```diff
@@ -46,11 +46,12 @@ def add_descriptions(self, target_item, descriptions, log):
             desc_content = descriptions[description]['value']
             lang = descriptions[description]['language']
             self.wdstuff.add_description(
-                lang, desc_content, target_item, summary=self.summary)
+                lang, desc_content, target_item)
             if log:
                 t_id = target_item.getID()
-                log.logit(t_id + " ADDED DESCRIPTION " +
-                          lang + " " + desc_content)
+                message = "{} ADDED DESCRIPTION {} {}".format(
+                    t_id, lang, desc_content)
+                log.logit(message)
 
     def add_aliases(self, target_item, aliases, log):
```

Comment:
Right. In order to force-mark something as an alias, one has to use [Monument.add_alias()](https://github.com/Vesihiisi/COH-tools/blob/master/importer/Monument.py#L137). I have combined add_labels and add_aliases, which checks for those and works on them separately using editAliases, and uses an edit summary that looks the same as when using addLabelOrAlias.